DeAceroDLabs/marketplace-frontend
# feat: [DMP-825] Added BACK_HOST and REACT_APP_ROOBYN_CATALOG_BASE_URL…

## What is done

[[DMP-825]  .env file](https://deacero.atlassian.net/browse/DMP-825)

## How is done

- REACT_APP_ROOBYN_CATALOG_BASE_URL, BACK_HOST variables are now required on .env file, this will help us to update only the env file when deploying to production or qa.
- The following variables also should change later when openpay is approved:
- REACT_APP_OPEN_PAY_MERCHANT_ID
- REACT_APP_OPEN_PAY_API_KEY
- These steps can not be done because of the dockerfile:
![image](https://user-images.githubusercontent.com/81311646/213313390-c3fd5b99-bdb2-4fdc-837e-4c6603774455.png)

![image](https://user-images.githubusercontent.com/81311646/213313496-b772089f-4b4c-4841-a8e3-0057a2a7fd6e.png)


## How to test

1. When in development add these 2 variables on the env file: 
REACT_APP_ROOBYN_CATALOG_BASE_URL=https://dev.api.catalog.roobyn.com
BACK_HOST=https://dev.api.ventaempleados.dlabs.com.mx
2. When in production add these 2 variables on the env file: 
REACT_APP_ROOBYN_CATALOG_BASE_URL=https://dev.api.catalog.roobyn.com
BACK_HOST=https://dev.api.ventaempleados.dlabs.com.mx
3. npm start
4. Everything should work as expected.

## Warning: Update .env variables on the dev VM before merging
